opencensus/trace/tracers/context_tracer.py

```diff
@@ -120,7 +120,8 @@ def end_span(self, *args, **kwargs):
         with self._spans_list_condition:
             if cur_span in self._spans_list:
                 span_datas = self.get_span_datas(cur_span)
-                self.exporter.export(span_datas)
+                for exporter in self.exporters:
+                    exporter.export(span_datas)
```

Comment:
If someone tries to send something to multiple exporters, but one is incorrectly configured, we should try export to the others still, right, are we just gonna hope that each `Exporter.emit` swallows the exceptions and doesn't raise any?

Reply:
I remember we had discussion about supporting multiple exporters and propagators, and people in that meeting generally agreed not to expose the interface for multiple exporters/propagators, but let people implement something like AggregatedExporter/Propagator. The idea is that you can implement a wrapper exporter that calls into multiple exporters, and you can decide what's the expected behavior - either abort the others if one failed, or continue.

Can we have a clarification here - do we want to have multiple exports/propagators interface for all languages? Or we want to stick with the aggregated approach?

Reply:
@reyang both Java and Go libraries support multiple destinations today. From a testing perspective or migration perspective this is a good feature to support. Of course, long term this could be handed off to the agent/collector, but requiring the agent/collector limits flexibility. I would propose supporting multiple destinations.

Reply:
@flands thank you! What about multiple propagators?

Reply:
IMO this is less of a concern -- switching and testing is straightforward and migration path is cutover. Let me check OC service team and get back shortly.

Reply:
@reyang I wasn't around for the discussion, can you link me? I don't think this is necessarily the best design, and I'm willing to believe we should add AggregatedExporter/Propagator classes instead.

@flands IMO if the API were closer to the [java client's](https://static.javadoc.io/io.opencensus/opencensus-api/0.17.0/io/opencensus/trace/export/SpanExporter.Handler.html) I think it would make sense to argue for a particular implementation, but as it is I think we have some flexibility here, and don't necessarily need to change the tracer.

Reply:
@c24t I couldn't find the discussion, it was Jul. or Aug. this year.

I remember at that time we were talking about how flush API should be provided, whether the user would need to call flush for each individual exporter (if we support multi), or a top level flush at tracer level should be provided (and should we block the flush API until all exporters have finished flush, what do we do if one exporter failed).

Reply:
Want to take the conversation around the design to #181?

Reply:
Yep! #181 seems to be a good place for discussion.